tests/device_tests/test_supershamir_reset_recovery.py

```diff
@@ -0,0 +1,286 @@
+import pytest
```

Comment:
Let's rename this file to test_shamir_reset_recovery_groups.py so it's clearer it relates to Shamir.

Reply:
done